src/ott/math/utils.py

```diff
@@ -37,6 +43,70 @@ def safe_log(  # noqa: D103
   return jnp.where(x > 0., jnp.log(x), jnp.log(eps))
 
 
+@functools.partial(jax.custom_jvp, nondiff_argnums=[1, 2, 3])
```

Comment:
To reduce this, I think we can do:
```python
norm = jax.custom_jvp(jax.jit(jnp.linalg.norm, static_argnames=("ord", "axis", "keepdims"), nondiff_argnums=[1, 2, 3])
```

Reply:
as discussed, will keep this, because of flexibility to have a custom pydocs.